packages/sdk/src/PromotionTiers.ts

```diff
@@ -21,6 +21,12 @@ export class PromotionTiers {
 	/**
 	 * @see https://docs.voucherify.io/reference/add-promotion-tier-to-campaign
 	 */
+	public get(promotionId: string) {
```

Comment:
Parameter name is misleading.
This endpoint is returning tier based on tierId - https://docs.voucherify.io/reference/get-promotion-tier
Please change parameter name to `tierId`